engine/src/main/java/org/hibernate/search/query/dsl/impl/ConnectedMultiFieldsRangeQueryBuilder.java

```diff
@@ -67,11 +67,11 @@ public Query createQuery() {
 			return queryCustomizer.setWrappedQuery( createQuery( fieldContexts.get( 0 ), conversionContext ) ).createQuery();
 		}
 		else {
-			BooleanQuery aggregatedFieldsQuery = new BooleanQuery();
+			BooleanQuery.Builder builder = new BooleanQuery.Builder();
```

Comment:
Got it. In previous commit c760655, there're same kind of naming problems. I'll apply the same role.